test/Microsoft.Azure.Devices.Edge.Test/EdgeAgentDirectMethods.cs

```diff
@@ -130,7 +130,7 @@ public async Task TestUploadModuleLogs()
         {
             string moduleName = "NumberLogger";
             int count = 10;
-            string sasUrl = "https://lefitcheblobtest1.blob.core.windows.net/upload-test?sv=2019-02-02&st=2020-08-03T17%3A14%3A16Z&se=2020-11-04T18%3A14%3A00Z&sr=c&sp=racwdl&sig=phKgqaaxSJTcZzUcggE%2FnhDljs4%2BhvCg7IOKk8iWTcY%3D";
+            string sasUrl = Environment.GetEnvironmentVariable("E2E_BLOB_STORE_SAS");
```

Comment:
Sorry I missed this earlier...`BlobStoreSas` should become a property on the `Context` object, rather than directly calling `Environment.GetEnvironmentVariable()` here.